golog.go

```diff
@@ -87,6 +87,15 @@ func SetOutputs(errorOut io.Writer, debugOut io.Writer) {
 	})
 }
 
+// RedirectOutputs temporarily redirects outputs. Returns a function that resets outputs to their prior settings.
+func RedirectOutputs(errorOut io.Writer, debugOut io.Writer) (reset func()) {
```

Comment:
Alternatively we could modify `SetOutputs` to return the reset function, which has no effect on existing code if they don't use the return value. But this is also LGTM

Reply:
Good idea, I'll do that!